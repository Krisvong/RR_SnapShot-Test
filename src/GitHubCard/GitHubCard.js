import React from 'react';
import Card from 'react-bootstrap/Card';
import SnackDeveloper from './SnackDeveloper.jpg'

const GitHubCard = () => {
  return (
    <Card>
      <Card.Img variant="top" src={SnackDeveloper} alt="Profile Pic" />
      <Card.Body>
        <Card.Title>Krisvong</Card.Title>
        <Card.Text>
          I am learning how to be a Software Developer,very slowly.
        </Card.Text>
      </Card.Body>
    </Card>
  );
};

export default GitHubCard;
