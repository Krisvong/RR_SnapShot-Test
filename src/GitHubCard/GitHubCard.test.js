import React from 'react';
import renderer from 'react-test-renderer';
import GitHubCard from './GitHubCard';

describe('GitHubCard', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<GitHubCard />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
