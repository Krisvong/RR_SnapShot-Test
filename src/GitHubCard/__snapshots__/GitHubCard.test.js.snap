// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <img
    alt="Profile Pic"
    className="card-img-top"
    src="SnackDeveloper.jpg"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      Krisvong
    </div>
    <p
      className="card-text"
    >
      I am learning how to be a Software Developer,very slowly.
    </p>
  </div>
</div>
`;
